.gitignore

```diff
@@ -23,3 +23,12 @@ venv/
 
 # CMake default directories
 build/
+
+# CMake user presets, generated by conan
+CMakeUserPresets.json
+
+# Exported compile commands
+compile_commands.json
+
+# Devcontainer setups
+.devcontainer
```

Comment:
No need to ignore .devcontainer anymore.

Reply:
Your argument is valid. On the other hand, if a user changes these files and wants them to be ignored, it is also possible to add these files to the user's local *.git/info/exclude* instead of the project-wide *.gitignore*. If the files are ignored via *.gitignore* then maintainers might overlook publishing updates to these files.